### ISSUE TITLE
*args is considered as missing in documentation

### ISSUE THREAD

**Issue by biomael** [PyCQA/pylint#3733]
Hi, I met an issue with the following code:

````
def function(file, *args, sep=";"):
    """Description of the function

    Parameters
    ----------
    file : str
        Path to the input.
    *args
        Relevant parameters.
    sep : str, optional
        File separator.
    """
````

### Current behavior
Return this warning: W9015: "args" missing in parameter documentation (missing-param-doc)

### Expected behavior
No warning must be returned as there is a documentation concerning *args

### pylint --version output
pylint 2.5.3
astroid 2.4.2
Python 3.7.2 (default, Feb 25 2019, 14:07:05)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]

**Comment by DanielNoord** [comment]
I'm able to reproduce this.

Adding `pylint.extensions.docparams` to `load-plugins` and `no-docstring-rgx=__.*__` makes this warning emit for me.

I have a feeling it might be because of the `*` because the message reports it is missing for `args` instead of `*args`. I will take a look!
